sensor/kubernetes/certrefresh/certrepo/service_certificates_repository.go

```diff
@@ -211,8 +214,8 @@ func (r *ServiceCertificatesRepoSecrets) createSecret(ctx context.Context, caPem
 		ObjectMeta: metav1.ObjectMeta{
 			Name:            secretSpec.SecretName,
 			Namespace:       r.Namespace,
-			Labels:          utils.GetSensorKubernetesLabels(),
-			Annotations:     utils.GetSensorKubernetesAnnotations(),
+			Labels:          utils.GetTLSSecretLabels(),
+			Annotations:     utils.GetSensorKubernetesLabels(),
```

Comment:
This should not be changed
```suggestion
			Annotations:     GetSensorKubernetesAnnotations(),,
```